### PR TITLE
Update nl.coffee

### PR DESCRIPTION
**Pull request by suzibol**
I added and changed some translations. I will work on this further in the coming days.

### CODE REVIEW

**Comment by SebastiaanBekker** [comment]
Hi @suzibol, thanks for the additions/updates. Saw one where i had a question: Why would you leave out Leraar in   update_account_remain_teacher: "Blijf Leraar"?

**Comment by Bryukh** [comment]
@suzibol Should I wait for fixing errors and commented lines?

**Comment by SebastiaanBekker** [comment]
@suzibol heb je hulp nodig bij het fixen? kan wel even kijken als je dat wil?

**Comment by suzibol** [comment]
Ik heb geen tijd gehad, zou erg fijn zijn! Het lukte me de vorige keer niet
om weer bij het bestand te komen...

Op za 29 sep. 2018 13:08 schreef Sebastiaan Bekker <notifications@github.com

File: app/locale/nl.coffee [hunk 1]
```diff
@@ -448,18 +448,18 @@ module.exports = nativeDescription: "Nederlands", englishDescription: "Dutch", t
...
 #    combo_challenge_unlocked: "Combo Challenge Unlocked"
 #    concept_challenge_unlocked: "Concept Challenge Unlocked"
 #    concept_challenge_complete: "Concept Challenge Complete!"
 #    combo_challenge_complete: "Combo Challenge Complete!"
-#    combo_challenge_complete_body: "Great job, it looks like you're well on your way to understanding __concept__!"
-#    replay_level: "Replay Level"
-#    combo_concepts_used: "__complete__/__total__ Concepts Used"
-#    combo_all_concepts_used: "You used all concepts possible to solve the challenge. Great job!"
-#    combo_not_all_concepts_used: "You used __complete__ out of the __total__ concepts possible to solve the challenge. Try to get all __total__ concepts next time!"
+     combo_challenge_complete_body: "Goed gedaan, het lijkt erop dat je het __concept__ bijna door hebt!"
```
Comment (SebastiaanBekker) [inline app/locale/nl.coffee]: denk dat hier een spatie teveel staat.

Comment (SebastiaanBekker) [reply app/locale/nl.coffee]: volgende wellicht ook

**Comment by SebastiaanBekker** [comment]
@Bryukh Hi, would like to help to get this branch live, but have no access to edit. Think it's a minor thing.... What's the best approach?

**Comment by suzibol** [comment]
I can do this next week, unfortunately no time now. Thanks a lot!

Op za 29 sep. 2018 19:20 schreef Sebastiaan Bekker <notifications@github.com

**Comment by suzibol** [comment]
> Hi @suzibol, thanks for the additions/updates. Saw one where i had a question: Why would you leave out Leraar in update_account_remain_teacher: "Blijf Leraar"?

I reviewed the file today, but I didn't find this section. What line number is it?

**Comment by Bryukh** [comment]
Thanks. Merged.

**Comment by suzibol** [comment]
Thanks. I hope you don't mind that I do it in batches. Probably a new one
sponsors.

Op do 4 okt. 2018 16:35 schreef Valentin Bryukhanov <
notifications@github.com>:

**Pull request merged**